lib/constants.js

```diff
@@ -1,17 +1,23 @@
 const BLOCKCHAIN = process.env.BLOCKCHAIN;
 const CONFIG_PATH = process.env.CONFIG_PATH;
+const START_BLOCK = parseInt(process.env.START_BLOCK || '0') - 1;
 const EXPORT_BLOCKS_LIST = process.env.EXPORT_BLOCKS_LIST || false;
+const PQUEUE_MAX_SIZE = parseInt(process.env.PQUEUE_MAX_SIZE || '100');
+const MAX_CONCURRENT_REQUESTS = parseInt(process.env.MAX_CONCURRENT_REQUESTS || '1');
```

Comment:
The `max size` one is for, how much tasks you could push into the queue. `max concurrent` one is for, how many tasks can work at the same time. You could have 100 of the first and 5 of the second

Reply:
I don't think you benefit from having two different sizes in your current design, because of how you queue new tasks - right away when the size drops. But anyways, don't see a harm in having both.